resources/content/schema/admin/admin-auth.json

```diff
@@ -159,35 +235,46 @@
         "publish.transitioningProgress": "cr",
 
         "resourceDefinition" : "r",
+        "resourceDefinition.name" : "r",
+        
+        "registrationToken" : "cr",
+        "registrationToken.state":"r",
+        "registrationToken.registrationUrl" : "r",
+        
+        "register": "cr",
+        "register.key":"cr",
+        "register.secretKey" : "r",
+        "register.accessKey" : "r",
+        
+        "schema.*": "r",
```

Comment:
Well, I meant a regex for the "Schema" resource type and all its fields to have a "r" permission.